sdktests/common_tests_events_contexts.go

```diff
@@ -227,7 +227,7 @@ func (c CommonEventTests) EventContexts(t *ldtest.T) {
 
 					if user := representContextAsOldUser(t, context); user != nil {
 						t.Run("with old user", func(t *ldtest.T) {
-							_ = basicEvaluateFlagWithOldUser(t, client, flagKey, user, defaultValue)
```

Comment:
Not related to this change, but this is some non-intuitive data flow.

Reply:
Agreed.